src/Stream/src/Abstractions/Attributes/OutputAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Steeltoe.Stream.Attributes
+{
+    /// <summary>
+    /// Indicates that an output binding target will be created by the framework.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Method, AllowMultiple = false)]
+    public class OutputAttribute : Attribute
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OutputAttribute"/> class.
+        /// </summary>
+        public OutputAttribute()
+        {
+            Name = null;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OutputAttribute"/> class.
+        /// </summary>
+        /// <param name="name">the binding target name</param>
+        public OutputAttribute(string name)
```

Comment:
Could combine the consturctors by using a null default on `name`
```suggestion
        public OutputAttribute(string name = null)
```

Reply:
Make change